source/digits_hits/include/GateJPETActor.hh

```diff
@@ -185,6 +208,9 @@ protected:
 	TFile * pFile;
```

Comment:
Probably it was in the old code, but are those field pFile and pListeVar, pMessenger initialized in the constructor?